assets/js/components/ClustersList.jsx

```diff
@@ -71,7 +95,23 @@ function ClustersList() {
         filterFromParams: true,
         filter: (filter, key) => (element) =>
           element[key].some((sid) => filter.includes(sid)),
-        render: (_, { sid }) => sid.join(', '),
+        render: (_, { sid }) =>
+          sid.map((singleSid) => {
+            const linkData = getSapSystemLinkBySID(
+              applicationInstances,
+              databaseInstances,
+              singleSid
+            );
+
+            return (
+              <SapSystemLink
+                systemType={linkData ? linkData.type : null}
+                sapSystemId={linkData ? linkData.sap_system_id : null}
+              >
+                {singleSid}
```

Comment:
What happens on multiple sids here?
Are they separated by spaces?

Reply:
I've used the tests to render how it would look like:

![Captura desde 2023-08-21 13-40-38](https://github.com/trento-project/web/assets/2668401/a83d2bb1-3b4f-4b17-a132-c5c5ff5a68e1)

![Captura desde 2023-08-21 13-50-23](https://github.com/trento-project/web/assets/2668401/78f408ee-9f4d-4fd7-a2f1-58c8fde86197)

Reply:
After speaking with @dottorblaster we decided it might look better if we add the coma back. @jagabomb wdyt?
![Captura desde 2023-08-21 15-37-22](https://github.com/trento-project/web/assets/2668401/d7733938-e282-4286-8e01-f0623287583d)